modoboa/lib/tests/test_cryptutils.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+"""Tests for crypto utility functions."""
+
+from __future__ import unicode_literals
+
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+from modoboa.lib import cryptutils
+
+
+class CryptUtilsTest(SimpleTestCase):
+
+    """Tests for modoboa.lib.cryptutils"""
+
+    def test_valid_secret_key_generated(self):
+        """Generate a 32 byte key"""
+        secret_key = cryptutils.random_key(32)
+        self.assertEqual(len(secret_key), 32)
+
+    @override_settings(SECRET_KEY="!8o(-dbbl3e+*bh7nx-^xysdt)1gso*%@4ze4-9_9o+"
```

Comment:
You could apply the override_settings decorator to the class instead of each method?

Reply:
fixed